test/plugin.test.js

```diff
@@ -96,6 +96,7 @@ QUnit.test('hls lifecyle', function(assert) {
 
 QUnit.test('dash lifecyle', function(assert) {
   let mediaPlayer = new MockMediaPlayer();
+  const oldBeforeInitialize = videojs.Html5DashJS.beforeInitialize;
```

Comment:
Is this set to anything here? May be worth setting it otherwise for the purpose of the test.